core/src/main/java/org/infinispan/commands/read/AbstractDataCommand.java

```diff
@@ -14,22 +16,33 @@
  */
 public abstract class AbstractDataCommand extends AbstractTopologyAffectedCommand implements DataCommand {
    protected Object key;
+   private long flags;
```

Comment:
the patch was: https://github.com/pruivo/infinispan/commit/c13741a94112656d3af4fa5a2bb399278b1670cc#diff-9afb10fb426aaf0f74720f940c36065bL15